ts/components/messages/MessageListItem.tsx

```diff
@@ -39,6 +44,15 @@ const UNKNOWN_SERVICE_DATA = {
 };
 
 class MessageListItem extends React.PureComponent<Props> {
+  get paymentExpirationInfo() {
+    return paymentExpirationInfo(this.props.message);
+  }
+  get paid(): boolean {
+    return this.props.payment !== undefined;
+  }
+  get isPaymentExpired() {
+    return this.paymentExpirationInfo.fold(false, info => isExpired(info));
```

Comment:
```suggestion
    return this.paymentExpirationInfo.fold(false,isExpired);
```
Perhaps other pieces of code in this file should be refactored like this 👍🏼